lib/l10n/intl_en.arb

```diff
@@ -1,6 +1,7 @@
 {
   "@@locale": "en",
-  "app_demo": "demo",
+  "app_demo": "Demo",
+  "app_demo_explanation": "This is a demo account. You may not be able to perform certain actions, like uploading files.",
```

Comment:
Fun fact: uploading files is the one write-operation you _can_ perform as a demo user: https://github.com/schul-cloud/schulcloud-server/blob/a2e15cb617916589cabbe18c0deea8499ed61080/backup/setup/roles.json#L267

Maybe keep it general like "This is a demo account. All actions that create or change data are deactivated and not visible." (translated from the web-client) and ignore that exception?

(This also applies to the German translation.)